README.md

```diff
@@ -57,28 +57,33 @@ Please read [CONTRIBUTING.md](https://github.com/benawad/dogehouse/blob/staging/
 
 ## DogeHouse Desktop
 
-A desktop app built with [electron](https://www.electronjs.org/) is available for Windows, Mac, and Linux
-
-Download links are in [the releases section](https://github.com/benawad/dogehouse/releases/latest)
-
-Dogehouse is also available in the AUR
-
-```bash
-yay -S dogehouse
-```
-
-And in an Ubuntu ppa
-
-```bash
-echo "deb http://ppa.dogehouse.tv/ ./" | sudo tee -a /etc/apt/sources.list > /dev/null
-wget -q -O - http://ppa.dogehouse.tv/KEY.gpg | sudo apt-key add -
-sudo apt-get update
-sudo apt-get install dogehouse
-```
+A desktop app built with [Electron](https://www.electronjs.org/) is available for Windows, Mac, and Linux.
+
+There are different ways to get the Electron desktop app:
+
+* Get the official builds from [here, in GitHub Releases][gh-releases]
+for any platform.
+* Get it from AUR for Arch/Manjaro with `yay -S dogehouse`.
+* Get the desktop client for Debian-based distros (including Ubuntu)
+from the official APT repo with these simple steps:
```

Comment:
remove the word "official" as this isn't official. add 3rd party and also add it to the AUR version.